TACC/Core-Portal
# task/FP-1769: Hide empty System Monitor if there are no systems to list

## Overview
Hide the System Monitor from the Dashboard instead of rendering an empty list of systems.

## Related

* [FP-1769](https://jira.tacc.utexas.edu/browse/FP-1769)

## Changes
Added a boolean value to the system monitor reducer's state so that we could check the if there are any systems to render.


## Testing

1. Navigate to the Dashboard, the system monitor should be displayed.
2. In your `settings_custom` file, set `_SYSTEM_MONITOR_DISPLAY_LIST` to an empty list.
3. Navigate to the Dashboard, the system monitor should be hidden.

## Notes

I chose this method to show/hide the System Monitor because simply checking the length of systems from the the system monitor list was triggering an infinite re-render cycle.
